documentation/user-guides/design-patterns/Design Pattern 1/Document_Field_Documentation.md

```diff
@@ -0,0 +1,19 @@
+# Pattern Name
+Document Field
+
+## Compentency Questions
+1) What document is this document field a part of?
+
+2) How many document fields are in the document?
+
+3) What is the content on these document fields?
```

Comment:
Relates to my comment on the SPARQL query. "the document" is used here and is presupposed to the document in question 2, but that isn't explicit enough.  A better style might be: given document field df1, what document d is df1 part of. 2) How many document fields are there in d. 3) What is the content of the document fields of document d.

Notice also varying language: "part of" is used in (1) "in" is used in (2). Suggest uniform.

The use of the word "on" in (3) seems to presuppose that you are looking at a concretization of d. Perhaps "associated with" might be more neutral.

Reply:
"content" is also very general and could equally apply to the ICEs. "What is the literal content of...".